runtime/near-vm-logic/src/mocks/mock_external.rs

```diff
@@ -211,6 +216,11 @@ impl External for MockedExternal {
     fn validator_total_stake(&self) -> Result<Balance> {
         Ok(self.validators.values().sum())
     }
+
+    #[cfg(feature = "protocol_feature_block_hash_host_fn")]
+    fn block_hash(&self, _block_height: BlockHeight) -> Result<Option<CryptoHash>> {
```

Comment:
As an aletrnaive, you can avoid conditional imports and use fully qualified names here:

```suggestion
    fn block_hash(&self, _block_height: near_primitives_core::types::BlockHeight) -> Result<Option<near_primitives::hash::CryptoHash>> {
```

Reply:
Done in https://github.com/near/nearcore/pull/4273/commits/a27e725e66f5c82b4f60635f3852b0f9a84d8c8e